src/main/scala/cromwell/engine/io/shared/SharedFileSystemIOInterface.scala

```diff
@@ -0,0 +1,42 @@
+package cromwell.engine.io.shared
+
+import java.nio.file.{Files, Paths}
+import cromwell.engine.io.gcs.{GoogleCloudStoragePath, GoogleCloudStorage}
+
+import scala.language.postfixOps
+
+import cromwell.binding.IOInterface
+
+import scala.util.Try
+
+class SharedFileSystemIOInterface extends IOInterface {
```

Comment:
You're not being consistent with IO. Down below you have `ioManager` treating the io as a single word and here IOInterface as two. I prefer the former (i.e. SharedFileSystemIoInterface) but choose one and stick to it